recipes/madminer/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "madminer" %}
+{% set version = "0.9.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/madminer-{{ version }}.tar.gz
+  sha256: 17eedd09da79db371530336904f248e8cde00032a53e50bbe36cbbb92f1c0939
+
+build:
+  # so use or statements in skip
+  # Will fail on Windows as PyTorch doe not support Windows, but failing on
+  # a noarch is preferred by the conda-forge maintainers
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  # Python 3.12+ is not allowed as uproot3_methods fails with
+  # AttributeError: 'FileFinder' object has no attribute 'find_module'.
+  # uproot3 and uproot3-methods are deprecated, so these should be removed
+  # in the future.
+  host:
+    - python >=3.8, <3.12
+    - pip
+  run:
+    - python >=3.8, <3.12
```

I've seen weird breakages b/c of spaces like this. In theory, it should not matter but better safe than sorry.

```suggestion
    - python >=3.8,<3.12
    - pip
  run:
    - python >=3.8,<3.12
```